.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,54 @@
+name: Benchmarks
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  unit_fluvio_protocol:
+    name: Fluvio Protocol - Unit Benchmarks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
```

Comment:
is still need to be nightly with criterion?

Reply:
Nope actually!